src/functionaltests/java/com/ericsson/ei/threadingAndWaitlistRepeat/ThreadingAndWaitlistRepeatSteps.java

```diff
@@ -19,25 +16,29 @@
 import org.springframework.core.env.Environment;
 import org.springframework.test.context.TestPropertySource;
 
+import com.ericsson.ei.handlers.EventToObjectMapHandler;
+import com.ericsson.ei.rules.RulesObject;
 import com.ericsson.ei.utils.FunctionalTestBase;
-import com.ericsson.ei.waitlist.WaitListWorker;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
 
-import cucumber.api.java.Before;
 import cucumber.api.java.en.Given;
 import cucumber.api.java.en.Then;
 
 @TestPropertySource(properties = { "threads.corePoolSize= 3", "threads.queueCapacity= 1", "threads.maxPoolSize= 4",
         "waitlist.collection.ttlValue: 60", "waitlist.initialDelayResend= 500", "waitlist.fixedRateResend= 1000",
         "spring.data.mongodb.database: ThreadingAndWaitlistRepeatSteps",
         "rabbitmq.exchange.name: ThreadingAndWaitlistRepeatSteps-exchange",
-        "rabbitmq.consumerName: ThreadingAndWaitlistRepeatStepsConsumer",
-        "logging.level.com.ericsson.ei.waitlist=DEBUG", "logging.level.com.ericsson.ei.handlers.EventHandler=DEBUG" })
+        "rabbitmq.consumerName: ThreadingAndWaitlistRepeatStepsConsumer", "logging.level.com.ericsson.ei.waitlist=OFF",
+        "logging.level.com.ericsson.ei.handlers.EventHandler=OFF" })
 
 @Ignore
 public class ThreadingAndWaitlistRepeatSteps extends FunctionalTestBase {
-
-    private File tempLogFile;
     private static final String EIFFEL_EVENTS_JSON_PATH = "src/functionaltests/resources/eiffel_events_for_thread_testing.json";
+    private static final String idRule = "{" + "\"IdRule\": \"meta.id\"" + "}";
```

Comment:
App private static finals should be un upper and snake case.

Reply:
done